packages/client/test/rpc/eth/call.spec.ts

```diff
@@ -10,48 +11,65 @@ import { checkError } from '../util'
 const method = 'eth_call'
 
 tape(`${method}: call with valid arguments`, async (t) => {
-  const blockchain = await Blockchain.create()
+  const blockchain = await Blockchain.create({ validateBlocks: false, validateConsensus: false })
```

Comment:
How come this passed before? It should throw on invalid PoW blocks right?

Reply:
before we were just using `vm.runTx`, but now are creating blocks and adding them to the blockchain (so our source code can setStateRoot properly), so needed these additional overrides